vite.config.ts

```diff
@@ -23,6 +23,7 @@ export default defineConfig(({ mode }) => ({
           '/code-of-conduct',
           '/terms-and-privacy',
           '/workshops',
+          '/workshops/angular-architects-signal-ddd-mfe',
           '/api/v1/workshops',
```

Comment:
try to add also this one:
          `'/api/v1/workshops/angular-architects-signal-ddd-mfe',`